plugins/input/bitbucket/server/server.go

```diff
@@ -0,0 +1,57 @@
+package server
+
```

Comment:
The bitbucket serve implementation looks partially done.  Wondering how similar the Bitbucket Cloud vs Bitbucket Server api interfaces are, and if you can combine them potentially.  All good if they're too different.

Reply:
Found that bitbucket-server uses an outdated version (v1.0) of their API while bitbucket-cloud uses v2.0.

There seems to be no Golang client for the old v1.0 API and the autogenerated client from swagger spec doesn't properly abstract over paginated results (like the repository lists).

So I figured if I'm gonna write code to account for pagination anyway, might as well write a bare HTTP approach for bitbucket-server instead of trying to use the client? Just for getting the repository lists. What do you think?